apps/frontend/src/components/ApproveRedirect.svelte

```diff
@@ -5,6 +5,13 @@
   import SuccessIcon from './base/svg/SuccessIcon.svelte';
   import ErrorIcon from './base/svg/ErrorIcon.svelte';
   import Loader from './base/Loader.svelte';
+  // import { Circle } from 'svelte-loading-spinners';
+  import Spinner from 'svelte-spinner';
+  let size = 25;
+  let speed = 750;
+  let color = '#A82124';
```

Comment:
We don't really have anywhere to store color's right now but this is coming soon in a style PR! Could you just put a `//TODO: Change to global color vairable. See <issue link here>` above and open a Github issue for having common style/ color coding throughout all components?